superset/assets/src/explore/components/ExploreViewContainer.jsx

```diff
@@ -299,14 +300,32 @@ class ExploreViewContainer extends React.Component {
 ExploreViewContainer.propTypes = propTypes;
 
 function mapStateToProps({ explore, charts, impressionId }) {
-  const form_data = getFormDataFromControls(explore.controls);
+  const rawFormDataControls = getControlsState(explore, explore.rawFormData);
+  const controls = Object.assign({}, explore.controls);
+  const formData = getFormDataFromControls(controls);
```

Comment:
I think you can just `getFormDataFromControls(explore.controls);` here and remove the previous line